src/screens/OrgList/OrganizationModal.spec.tsx

```
@@ -0,0 +1,486 @@
+              setFormState={mockSetFormState}
+              createOrg={mockCreateOrg}
+              t={(key) => key}
+              tCommon={(key) => key}
+              userData={undefined}
+            />
+          </I18nextProvider>
+        </BrowserRouter>
+      </Provider>,
+    );
+    setup();
+    expect(screen.getByTestId('modalOrganizationHeader')).toBeVisible();
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)